.travis.yml

```diff
@@ -14,6 +14,7 @@ before_install:
 before_script:
 - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
 - chmod +x ./cc-test-reporter
+- "cp spec/dummy/config/database.yml.sample spec/dummy/config/database.yml"
 - "./cc-test-reporter before-build"
```

Comment:
These 3 statements at the top here are a unit for cc-test-reporter, so this new line should not interleave them

Reply:
Do I not need it at all? Or does this build not make use of `DATABASE_URL` and it is still needed?

Reply:
yeah, you don't need it at all